modules/tinymce/src/themes/silver/test/ts/browser/sidebar/SidebarShowOptionTest.ts

```diff
@@ -73,6 +77,7 @@ describe('browser.tinymce.themes.silver.sidebar.SidebarShowOptionTest', () => {
       sidebar_show: 'sidebartwo'
     });
     assertNotAnimating();
+    assertSlotContainerNotOpen();
```

Comment:
This doesn't look right? Based on the test `sidebartwo` should be open here.
```suggestion
    assertSlotContainerOpen();
```